pyproject.toml

```diff
@@ -41,6 +41,7 @@ dependencies = [
     "pillow",
     "h11>=0.16.0",
     "kubernetes",
+    "sqlalchemy[asyncio]>=2.0.41",
```

Comment:
should we only add this dependency if a distro actually includes a sqlalchemy store?